vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -278,3 +248,55 @@ func getContainer(containerName string, pod *apiv1.Pod) *apiv1.Container {
 	}
 	return nil
 }
+
+// applyLimitRange updates recommendation if recommended resources are outside of limits defined in VPA resources policy
+func applyLimitRange(recommendation apiv1.ResourceList, container apiv1.Container, limitRange *apiv1.LimitRangeItem) []string {
```

Comment:
applyPodLimitRange?

Reply:
I'm confused, should it be pod LimitRange or container LimitRange?

Reply:
Container limit range, sorry for the confusion.

Reply:
Done